classes/student.rb

```diff
@@ -1,6 +1,6 @@
 class Student < Person
-  def initialize(classroom, age, name)
-    super(age, name)
+  def initialize(classroom, age, name, parent_permission)
+    super(age, name, parent_permission: parent_permission)
```

Comment:
- [ ] Kindly make sure to implement the changes asked by the previous reviewer. Follow below:
```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
end
```